src/skins/movistar.tsx

```diff
@@ -139,7 +139,7 @@ export const getMovistarSkin: GetKnownSkin = (variant) => {
 
             // GLOBAL
             brand: palette.movistarBlue,
-            brandDark: palette.movistarBlue55,
+            brandHigh: palette.movistarBlue55,
             inverse: palette.white,
             neutralHigh: palette.grey6,
             neutralMedium: palette.grey5,
```

Comment:
neutralLow wrong color, change to `grey1`
https://github.com/Telefonica/mistica-design/blob/f823f0765e6e8491e883a2e7019a736e1206a359/tokens/movistar.json#L299